include/toml/toml.h

```diff
@@ -140,12 +146,15 @@ class Value {
     const Value* find(size_t index) const;
     Value* find(size_t index);
     Value* push(const Value& v);
+    Value* push(Value && v);
 
     // ----------------------------------------------------------------------
     // Others
 
     // Writer.
-    void write(std::ostream*, const std::string& keyPrefix = std::string()) const;
+    std::string getIndent(int indent) const;
```

Comment:
This should be a static method, since this doesn't depend on any instance fields.